js/NarrativeView.js

```diff
@@ -248,39 +248,35 @@ class NarrativeView extends ComponentView {
   setupBackNextLabels(index = this.model.getActiveItem().get('_index')) {
     const totalItems = this.model.getChildren().length;
 
-    const isAtStart = index === 0;
-    const isAtEnd = index === totalItems - 1;
+    const _isAtStart = index === 0;
+    const _isAtEnd = index === (totalItems - 1);
 
-    const globals = Adapt.course.get('_globals');
-    const narrativeGlobals = globals._components._narrative;
+    const _globals = Adapt.course.get('_globals');
+    const narrativeGlobals = _globals._components._narrative;
 
-    let prevTitle = isAtStart ? '' : this.model.getItem(index - 1).get('title');
-    let nextTitle = isAtEnd ? '' : this.model.getItem(index + 1).get('title');
+    const prevItem = !_isAtStart && this.model.getItem(index - 1);
+    const nextItem = !_isAtEnd && this.model.getItem(index + 1);
 
-    let backItem = isAtStart ? null : index;
-    let nextItem = isAtEnd ? null : index + 2;
+    const prevTitle = prevItem ? prevItem.get('title') : '';
```

Comment:
```suggestion
    const prevTitle = prevItem?.get('title') ?? '';
```
This should also work. If no prevItem then the left side of the ?? operator is undefined causing it to use the right side

Reply:
Can you remove all of the blank lines from this section please? It makes the file unnecessarily long having odd white space inside functions. 250,253,256,259,262 at least

Reply:
Done. I had to rework prevItem and nextItem since they were returning false rather than null. Otherwise, using the optional chaining operator didn't work.